governance.md

```diff
@@ -186,8 +184,23 @@ Code of Conduct) and not other technical reasons (e.g. we have different
 viewpoints on project direction) or personal reasons (e.g. they belong to a
 competitor's organization). We ask that a majority of Steering Council members
 respond for quorum, and that those votes come from people employed/associated
-with at least three institutions.
+with at least two institutions.
+
+### Emeritus council members
+
+When a member leaves the Council due to inactivity they automatically become an
+Emeritus Council Member where they can continue to provide the same expert guidance
+to the community but without the Ownership status on GitHub or any voting rights.
 
+Members that are ejected by council vote for harmful behaviour they will not be given
```

Comment:
Small typo

```suggestion
Members that are ejected by council vote for harmful behaviour will not be given
```

Also not sure where we land on the spelling of "behavior" vs "behaviour" :).